Delete intersphinx, delete entry to set sphinxjp.themecore which is absent from portage
diff -ur numba-0.19.2.orig/docs/source/conf.py numba-0.19.2/docs/source/conf.py
--- docs/source/conf.py	2015-06-28 15:19:14.573386743 +0800
+++ docs/source/conf.py	2015-06-28 15:20:16.808191183 +0800
@@ -36,12 +36,9 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx.ext.intersphinx',
     'sphinx.ext.todo',
     #'sphinx.ext.mathjax',
     'sphinx.ext.autodoc',
-    #'sphinx.ext.graphviz',
-    'sphinxjp.themecore',
 ]
 
 todo_include_todos = True
